core/urls.py

```diff
@@ -1,10 +1,11 @@
 from django.urls import path
 
-from .views import PostDetailView, PostListView, PostSaveView, PostVoteView
+from .views import AddPostView, PostDetailView, PostListView, PostSaveView, PostVoteView
 
 urlpatterns = [
         path("post-list/", PostListView.as_view(), name="post-list"),
         path("post-detail/<int:pk>/", PostDetailView.as_view(), name="post-detail"),
+        path("add-post/", AddPostView.as_view(), name="add-post"),
```

Comment:
Maybe it's better to stay with CRUD naming convention: CreatePostView

Reply:
Yes, please.

Reply:
done